lib/src/executable.dart

```diff
@@ -12,12 +12,11 @@ import 'json_utils.dart';
 /// typing <name> executes bin/<script>.dart.
 
 class Executable extends Jsonable {
-  // String name;
+  String name;
```

Comment:
Just change script to a non-nullable and handle everything that way if you think it's necessary. `name` is just extra stuff.

Reply:
name and script can be different and having both in the class seems to be handy rather than requiring the user to have access to the key to get the name.

Reply:
This changes the behavior of the constructor across multiple different projects for one. `script` already suffices and yaml allows for nulls, so the behavior is consistent there. Users should just account for scenarios where there is no string provided or we should just force the behavior regardless. I don't actually know your specific use case, but this seems bad.